canonical/ubuntu.com
# make event avatars look like live and fix-up old events on /cloud and…

## Done
- removed background squircle from the map event avatars
- moved css into local as it was in two sections
- updated the out of date events on /cloud
- removed the out of date event on /internet-of-things/partners and replaced with the default
## QA
1. go to /cloud and see that the squircle is gone
2. see that the events are current
3. go to /internet-of-things and see tha the expired event has been replaced with a promo for developers
## Issue / Card

Closed #222
